topology/graph/traversal/traversal.go

```diff
@@ -354,6 +354,14 @@ func Lte(s interface{}) *LTEGraphElementMatcher {
 	return &LTEGraphElementMatcher{value: s}
 }
 
+// ForeverGraphElementMatcher descripes a enitire time limit in the history
+type ForeverGraphElementMatcher struct{}
```

Comment:
It's a bit different that the other GraphElementMatcher. Maybe `ForeverConstant` or `ForeverPredicate`